install.sh

```diff
@@ -194,94 +195,95 @@ __install_installer_deps() {
+		sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9373E47F9BF8216D23D32BBBE1E6759023F386C7
+		__add_deb_repo "deb http://ppa.launchpad.net/securityonion/stable/ubuntu xenial main" \
+		"securityonion-ubuntu-stable-xenial"
+		__install_packages securityonion-bro
+		ln -s /opt/bro /opt/zeek
 	else
 		case "$_OS" in
 			Ubuntu)
-				__add_deb_repo "deb http://download.opensuse.org/repositories/home:/logan_bhis:/branches:/network:/bro-2-6/xUbuntu_$(lsb_release -rs)/ /" \
-					"Bro" \
-					"https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro-2-6/xUbuntu_$(lsb_release -rs)/Release.key"
+				__add_deb_repo "deb [ arch=$(dpkg --print-architecture) ] http://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/ /" \
+				"security:zeek" \
+				"https://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/Release.key"
 				;;
 			CentOS|RedHatEnterprise|RedHatEnterpriseServer)
-				__add_rpm_repo https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro-2-6/CentOS_7/home:logan_bhis:branches:network:bro-2-6.repo
+				__add_rpm_repo "https://download.opensuse.org/repositories/security:zeek/CentOS_7/security:zeek.repo"
 				;;
 		esac
+		__install_packages zeek-lts
 	fi
 
-	__install_packages bro broctl
 
-	if [ -d /opt/bro/logs/ ]; then		#Standard directory for Bro logs when installed by Rita
-		chmod 2755 /opt/bro/logs
-	elif [ -d /var/log/bro/ ]; then		#Standard directory for Bro logs when installed by apt...
-		mkdir -p /opt/bro/logs/		#...and we move the log storage over to /opt/bro/logs so we can have one place to mount external storage.
-		chmod 2755 /opt/bro/logs
-		mv -f /var/log/bro /var/log/bro.orig
+	if [ -d /opt/zeek/logs/ ]; then		#Standard directory for Zeek logs when installed by Rita
+		chmod 2755 /opt/zeek/logs
+	elif [ -d /var/log/zeek/ ]; then		#Standard directory for Zeek logs when installed by apt...
+		mkdir -p /opt/zeek/logs/		#...and we move the log storage over to /opt/zeek/logs so we can have one place to mount external storage.
+		chmod 2755 /opt/zeek/logs
+		mv -f /var/log/zeek /var/log/zeek.orig
 		cd /var/log
-		ln -s /opt/bro/logs bro
+		ln -s /opt/zeek/logs zeek
 		cd -
 	fi
-	_BRO_PKG_INSTALLED=true
-	_BRO_INSTALLED=true
-	_BRO_PATH="/opt/bro/bin"
+	_ZEEK_PKG_INSTALLED=true
+	_ZEEK_INSTALLED=true
+	_ZEEK_PATH="/opt/zeek/bin"
 }
 
 __install_ja3() {
-	local_path=$_BRO_PATH/../share/bro/site/
+	local_path=$_ZEEK_PATH/../share/zeek/site/
 
 	mkdir -p $local_path/ja3/
 
-	for one_file in __load__.bro intel_ja3.bro ja3.bro ja3s.bro ; do
+	for one_file in __load__.zeek intel_ja3.zeek ja3.zeek ja3s.zeek ; do
 		if [ ! -e $local_path/ja3/$one_file ]; then
-			curl -sSL "https://raw.githubusercontent.com/salesforce/ja3/cb29184df7949743c64fcb190c902dfe72523e38/bro//$one_file" -o "$local_path/ja3/$one_file"
+			curl -sSL "https://raw.githubusercontent.com/salesforce/ja3/master/zeek/$one_file" -o "$local_path/ja3/$one_file"
```

Comment:
If the `securityonion-bro` package isn't zeek v3 or higher I don't think this will work for it so we should double-check.

Reply:
If `securityonion-bro` isn't higher than v3, `__install_ja3` won't run anyway. It's considered an incompatible version based on this line [here](https://github.com/activecm/rita/blob/524-convert-to-zeek/install.sh#L490). We could instead ask the user if they would like us to update `securityonion-bro`? Actually, that may be caught anyway by the initial `apt update & apt upgrade`...